### ISSUE TITLE
Trouble reading in DHS data

### ISSUE THREAD

**Issue by emmab725** [GIS4DEV/RP-Malcomb#1]
Hey everyone, I tried to run the R script however ran into an issue when downloading the DHS data. This is the error I'm getting:
Error in names(filedatatypelist_DHS) <- paste0("filedatatypelist_", qdapRegex::rm_between(filedatatypelist_DHS_line, : 'names' attribute [1] must be the same length as the vector [0]
Anyone know what this means?

**Comment by josephholler** [comment]
Same issue I saw with Maddie and Jackson, still unresolved: https://github.com/GIS4DEV/GIS4DEV.github.io/issues/38#

**Comment by emmab725** [comment]
Hey all, I was able to find a workaround because I had the dhs data downloaded already from last week. So rather than running the following code (around line 84 depending on how you commented your script):
dhs_downloads = get_datasets(
  c("MWHR61SV", "MWGE62FL", "MWHR4ESV", "MWGE4BFL"),
  all_lower = FALSE,
  download_option = "rds"

I just called directly to the data through the folder structure it was saved in. 

So instead of running 
dhsclusters_2010 = readRDS(dhs_downloads$MWGE62FL),
I ran:
dhsclusters_2010 = readRDS(here("data", "raw", "private", "datasets", "MWGE62FL.rds")) 

and instead of:
dhshh_2010 = readRDS(dhs_downloads$MWHR61SV) %>% zap_labels() 
I ran:
dhshh_2010 = readRDS(here("data", "raw", "private", "datasets", "MWHR61SV.rds")) %>% zap_labels() 

because the data was stored in data/raw/private/datasets. That way, we can skip the first chunk of code where we created dhs_downloads altogether. 

Hope this helps anyone who had the same issue!
